src/EntityFramework6.Npgsql/SqlGenerators/VisitedExpression.cs

```diff
@@ -526,7 +526,7 @@ internal override void WriteSql(StringBuilder sqlText)
             }
             else
             {
-                var wrap = !(_from is LiteralExpression || _from is ScanExpression);
+                var wrap = !(_from is LiteralExpression || _from is ScanExpression || _from is FunctionExpression);
```

Comment:
@rwasef1830, yes, I think this is necessary because of Postgres syntax. In example, the following works:
`select * from "dbo"."GetBlogsByName"('some')` 
but this one does not: 
`select * from ("dbo"."GetBlogsByName"('some'))`